documentation/library_guide/extension_api.rst

```diff
@@ -240,9 +240,10 @@ to be called on both host and device.
 
 The following algorithms are available to use with the ranges:
 
-``for_each``, ``copy``, ``transform``, ``find``, ``find_if``, ``find_if_not``, ``find_end``, ``find_first_of``, ``search``, ``is_sorted``,
-``is_sorted_until``, ``reduce``, ``transform_reduce``, ``min_element``, ``max_element``, ``merge``, ``minmax_element``,
-``exclusive_scan``, ``inclusive_scan``, ``sort``, ``stable_sort``, ``transform_exclusive_scan``, ``transform_inclusive_scan``.
+``all_of``, ``any_of``, ``copy``, ``count``, ``count_if``, ``equal``, ``exclusive_scan``, ``find``, ``find_if``, ``find_if_not``, ``find_end``,
+``find_first_of``, ``for_each``, ``inclusive_scan``, ``is_sorted``, ``is_sorted_until``, ``min_element``, ``max_element``, ``merge``,
+``minmax_element``, ``move``, ``reduce``, ``remove``, ``remove_if``, ``replace``, ``search``, ``sort``, ``stable_sort``, ``transform``,
```

Comment:
No, I cannot fix it, because it is not an error :-) To add - I can.
Furthermore, according to #199 we have not announced `replace_if``....
But, as far as there is the implementation of one (and the test), I do not mind to add `replace_if`` to the docimentation.